package.json

```diff
@@ -6,14 +6,11 @@
   "main": "base-theme/assets/index.js",
   "scripts": {
     "lint": "eslint $PROJECT_CWD/'**/*.{js,jsx,ts,tsx}'",
-    "start": "ts-node ./package_scripts/start.ts",
```

Comment:
Apparently I accidentally committed this in #964. We did not notice because previously `yarn start` didn't really do anything.